### PR TITLE
Feature/us1725 fees for deposit

### PR DESCRIPTION
**Pull request by IngageRitterS**

### CODE REVIEW

File: Gateway/crds-angular/Services/StripeEventService.cs [hunk 1]
```diff
@@ -153,6 +153,8 @@ public TransferPaidResponseDTO TransferPaid(DateTime? eventTimestamp, StripeTran
                 DepositName = batchName,
                 // This is the amount from Stripe - will show out of balance if does not match batch total above
                 DepositTotalAmount = transfer.Amount / 100M,
+                ProcessorFeeTotal = transfer.Fee / 100M,
+                DepositAmount = (transfer.Amount/100M - transfer.Fee/100M),
```
Comment (IngageBrewerM) [inline Gateway/crds-angular/Services/StripeEventService.cs]: Should we consider removing this *magic number* and creating a named constant to describe what this value is?  Lets ask the team.

File: Gateway/Crossroads.Utilities/CrossroadsConstants.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿
+namespace Crossroads.Utilities 
+{
+    public static class Constants
+    {
+        public static decimal StripeDecimalConversionValue =  100M;
```
Comment (dtkocher) [inline Gateway/Crossroads.Utilities/CrossroadsConstants.cs]: I am not doing the actual code review just glancing at stuff.  Should this be STRIPE_DECIMAL_CONVERSION_VALUE

Comment (IngageBrewerM) [inline Gateway/Crossroads.Utilities/CrossroadsConstants.cs]: :+1:

Comment (IngageKrizJ) [inline Gateway/Crossroads.Utilities/CrossroadsConstants.cs]: Use const instead of static in .Net, right?

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**